github/tests/ReplayData/Issue606.testProjectList.txt

```diff
@@ -0,0 +1,55 @@
+https
```

Comment:
Does the ReplayData let you run my tests? Or do you need to record your own data?

Reply:
Understood now - anyone can run my recorded test.